### DISCUSSION TITLE
Getting this unicodeEncode Error for few sites:

### DISCUSSION THREAD

**Discussion by suri02** [Cmastris/robotstxt-change-monitor#1] [Q&A] [answered]
Hi,

I tried to run few websites i'm getting the below error for 2 websites. Can anyone please help me on this how to resolve this. 

16-02-21, 11:09: Error: https://www.example.com/. Unexpected error during https://www.example.com/ check.
TYPE: <class 'UnicodeEncodeError'>
DETAILS: 'charmap' codec can't encode character '\u0432' in position 3529: character maps to <undefined>
TRACEBACK:

File "D:\Python\robots_txt\main.py", line 222, in run_check
    self.update_records(extraction)
  File "D:\Python\robots_txt\main.py", line 313, in update_records
    new.write(new_extraction)
  File "C:\Python34\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]

Thank you,

**Comment by Cmastris** [comment, marked as answer]
Just noting that this is being addressed in #2 🙂 

Edit: Now fixed.
